zerver/tests/test_custom_profile_data.py

```diff
@@ -80,14 +80,18 @@ def test_create(self) -> None:
         self.assertEqual(field.id, field.order)
 
         data["name"] = "Name "
-        data["hint"] = "Some name"
+        data["hint"] = "@**iago**"
         data["field_type"] = CustomProfileField.SHORT_TEXT
         data["display_in_profile_summary"] = "true"
         result = self.client_post("/json/realm/profile_fields", info=data)
         self.assert_json_success(result)
 
         field = CustomProfileField.objects.get(name="Name", realm=realm)
+        rendered_name = "<p>Name</p>"
+        rendered_hint = '<p><span class="user-mention" data-user-id="11">@Iago</span></p>'
         self.assertEqual(field.id, field.order)
+        self.assertEqual(field.rendered_name, rendered_name)
+        self.assertEqual(field.rendered_hint, rendered_hint)
```

Comment:
We should test this for updating fields as well.

Reply:
And for default external fields as well. Having these tests would be good to make sure that we do not accidentally remove those lines to set the `rendered_..` fields.